src/pages/library/common.js

```diff
@@ -48,6 +48,44 @@ const groupByFeaturedTags = (workspaces, sidebarSections) => _.flow(
   _.fromPairs
 )(sidebarSections)
 
+const numLabelsToRender = 5
```

Comment:
One of my college professors suggested "count" for size types, making this `labelsToRenderCount`. I think I like `numLabelsToRender` better.

Reply:
+1 for `numLabelsToRender`